SwiftCharts/Chart.swift

```diff
@@ -8,32 +8,71 @@
 
 import UIKit
 
+/// ChartSettings allows configuration of the visual layout of a chart
 public class ChartSettings {
+
+    /// Empty space in points added to the leading edge of the chart
     public var leading: CGFloat = 0
+
+    /// Empty space in points added to the top edge of the chart
     public var top: CGFloat = 0
+
+    /// Empty space in points added to the trailing edge of the chart
     public var trailing: CGFloat = 0
+
+    /// Empty space in points added to the bottom edge of the chart
     public var bottom: CGFloat = 0
+
     public var labelsSpacing: CGFloat = 5
```

Comment:
/// The spacing in points between axis labels when using multiple labels for each axis value (currently only used in x axis)